tests/foreman/ui/test_registration.py

```diff
@@ -76,23 +75,33 @@ def test_positive_org_loc_change_for_registration(
 
     :CaseImportance: Medium
     """
-    new_org = target_sat.api.Organization().create()
-    new_loc = target_sat.api.Location().create()
-    target_sat.api.ActivationKey(organization=new_org).create()
-    with target_sat.ui_session() as session:
-        session.organization.select(org_name=module_org.name)
+    new_org = module_target_sat.api.Organization().create()
+    new_loc = module_target_sat.api.Location().create()
+    ak = module_target_sat.api.ActivationKey(organization=new_org).create()
+    with module_target_sat.ui_session() as session:
+        session.organization.select(org_name=module_sca_manifest_org.name)
         session.location.select(loc_name=module_location.name)
-        cmd = session.host.get_register_command()
+        cmd = session.host.get_register_command(
+            {
+                'general.activation_keys': module_activation_key.name,
+                'general.insecure': True,
+            }
```

Comment:
As there'll be single AK in org and loc, and it'll be auto-populated, so we can skip this
```suggestion

```

Reply:
It's the same reason as given below. The page doesn't refresh on selecting org and loc so activation key doesn't get auto-populated resulting in failure.